src/main/java/com/ea/restaurant/exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.ea.restaurant.exceptions;
+
+public class UserNotFoundException extends RuntimeException {
+  public UserNotFoundException(String username) {
```

Comment:
Can you reuse  `EntityNotFoundException`?

Reply:
Deleted. It will be implemented later.

Reply:
Done.